selecta

```diff
@@ -276,6 +266,15 @@ class Search
     end
   end
 
+  def move(direction)
```

Comment:
I didn't even mean the constants/enums (which are less ambiguous IMO), but the methods.